agents/ppo.py

```diff
@@ -42,48 +48,57 @@ def create_from_config(cls, config):
             critic_loss_weight=config.get('critic_loss_weight', 1.0)
         )
 
-    def forward(self, state: State) -> Tuple[State, Policy, Value, Value]:
-        next_state, policy = self._actor.forward(state)
-        curr_v_func = self._critic.forward(state)  # (batch_size)
-        q_estimate = self._critic.forward(next_state)  # (batch_size)
-        # TODO [Vladimir Baikalov]: make same inerface for every agent's forward function
-        return next_state, policy, curr_v_func, q_estimate
+    def forward(self, inputs: Dict[str, Any]):
+        ppo_input = inputs[self._prefix]
+        next_state, policy = self._actor.forward(**ppo_input)
+        curr_v_func = self._critic.forward(**ppo_input)  # (batch_size)
+        q_estimate = self._critic.forward(**{**ppo_input, 'state': next_state})  # (batch_size)
+        self._memory.append({
+            'state': ppo_input,
+            'policy': policy,
+            'curr_v_func': curr_v_func,
+            'q_estimate': q_estimate
```

Comment:
If you really want to get Q-value, it is computed in a different way. It can be derived from V-function but these values are not equal